src/sql/base/browser/ui/checkbox/checkbox.ts

```diff
@@ -15,9 +16,14 @@ export interface ICheckboxOptions {
 	ariaLabel?: string;
 }
 
+export interface ICheckBoxStyles {
+	disabledCheckboxForeground?: Color;
```

Comment:
looks like casing of "Checkbox" and "CheckBox" is inconsistent between line 19 and 20.  We should probably be consistent here.

Reply:
good catch.